tests/unit_tests/test_game.py

```diff
@@ -150,6 +150,35 @@ async def test_ffa_not_rated(game, game_add_players):
     assert game.validity == ValidityState.FFA_NOT_RANKED
 
 
+async def test_generated_map_is_rated(game, game_add_players):
+    game.state = GameState.LOBBY
+    game.map_file_path = 'maps/neroxis_map_generator_1.0.0_1234.zip'
+    game_add_players(game, 2, team=1)
+    await game.launch()
+    await game.add_result(0, 1, 'victory', 5)
+    game.launched_at = time.time() - 60 * 20    # seconds
+    await game.on_game_end()
+    assert game.validity == ValidityState.VALID
+
+
+async def test_unranked_generated_map_not_rated(database, game, game_add_players):
+    async with database.acquire() as conn:
+        await conn.execute(
+            "INSERT into map_version (id, max_players, "
+            "width, height, version, filename, ranked, hidden, map_id)"
+            "VALUES (0, 4, 512, 512, 1, \"maps/neroxis_map_generator_sneaky_map.zip\","
+            "0, 0, 0)"
+        )
```

Comment:
Don't insert in the test, add it to `tests/data/test-data.sql`.

Reply:
Will do